build.sbt

```diff
@@ -32,13 +34,16 @@ assemblyJarName in assembly := s"${name.value}-uber.jar"
 
 assemblyMergeStrategy in assembly := {
   case "META-INF/io.netty.versions.properties" => MergeStrategy.last
+  // engine uses bblfsh client scala, which also uses scalapb but different version
```

Comment:
Wonder if there is a reason we could not declare, document and use same version of scalapb as bblfsh client does?

Reply:
1. bblfsh client-scala uses quite old version. We use last version 0.7.1. It contains bug fixes and other improvements.
2. I believe it's better to update bblfsh client-scala then stick to old version. Temporary we can live with custom merge strategy for proto files (they aren't needed after scala files are generated so here it could be removed strategy as well)
3. Updating version in bblfsh client-scala is a bit tricky because master isn't compiling right now (I'll take a look why today)